tests/src/Nightwatch/Tests/EditMostUsedContent.js

```diff
@@ -0,0 +1,59 @@
+/**
+ * @file
+ * Testing of editing for most used bundle type.
+ */
+
+// eslint-disable-next-line import/no-dynamic-require
+const apm = require(`${process.cwd()}/node_modules/elastic-apm-node`);
+const thunderUtils = require("../utils");
+
+module.exports = {
+  "@tags": ["Thunder", "Thunder_Base_Set"],
+  before(browser, done) {
+    browser.apm = apm;
+
+    // Get site information.
+    thunderUtils.setSiteInfo(
+      "test-admin",
+      "test-admin",
+      {
+        rule: "count",
+        index: 0
+      },
+      browser,
+      done
+    );
+  },
+  editMostUsedContent(browser) {
+    const { bundle, required_fields: requiredFields } = browser._site_info;
+
+    browser
+      .resizeWindow(1024, 1024)
+      .performance.startMeasurement(
+        process.env.THUNDER_APM_URL,
+        "Edit most used content (min)",
+        `.${process.env.THUNDER_SITE_HOSTNAME}`
+      )
+      .drupalLogin({ name: "test-admin", password: "test-admin" })
+      .useXpath()
+      .drupalRelativeURL(`/admin/content?type=${bundle}`)
+
+      .performance.startMark("full task")
+      .click(
+        '(//li[contains(@class,"dropbutton-action")])[1]//a[contains(@href, "edit")]'
+      )
+      .waitForElementVisible('//*[@id="edit-submit"]', 1000)
+      .autoFillFields(requiredFields)
```

Comment:
It would be nice to measure the loading time of form.

And also it would be nice to measure `autoFillFields` (wrap it in performance mark). In that case, it's easier to drill-down the problematic part, if some performance drop appears.